include/deal.II/lac/la_vector.templates.h

```diff
@@ -26,6 +26,51 @@ DEAL_II_NAMESPACE_OPEN
 namespace LinearAlgebra
 {
   template <typename Number>
+  void Vector<Number>::reinit(const size_type size,
+                              const bool omit_zeroing_entries)
+  {
+    ReadWriteVector<Number>::reinit(size, omit_zeroing_entries);
```

Comment:
`Vector` derives from both `ReadWriteVector` and `VectorSpaceVector` and this PR also add a `reinit` function to `VectorSpaceVector`.